Sources/libhostmgr/CLI/Console.swift

```diff
@@ -1,5 +1,6 @@
 import Foundation
 import ConsoleKit
+import tinys3
```

Comment:
Is `tinys3` used in this file?

Reply:
Yep! Just for [this helper](https://github.com/Automattic/hostmgr/blob/39f519bf55c99ed5ced7ab23d666197c8bb57834/Sources/libhostmgr/CLI/Console.swift#L142)